sdk/compute/Microsoft.Azure.Management.Compute/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("42.0.0.0")]
-[assembly: AssemblyFileVersion("42.0.0.0")]
+[assembly: AssemblyVersion("43.0.0.0-preview.1")]
```

Comment:
Generally we don't do this witl assembly file versions.  This should be 43.0.0.1

Reply:
@markcowl , edited as you wanted, please re-review.